lib/beacon/router.ex

```diff
@@ -153,6 +156,19 @@ defmodule Beacon.Router do
     end
   end
 
+  @doc """
+  Creates a beacon sitemap index.
+  """
+  defmacro beacon_sitemap_index(filename) do
+    quote bind_quoted: binding(), location: :keep do
+      import Phoenix.Router, only: [scope: 3, get: 3, get: 4]
```

Comment:
Will `get: 3` be used here?

```suggestion
      import Phoenix.Router, only: [scope: 3, get: 4]
```